### ISSUE TITLE
Error in Ballot Contract

### ISSUE THREAD

**Issue by 43061b4a** [ethereum/solidity#3061]
Sample Ballot contract on [this](https://solidity.readthedocs.io/en/latest/solidity-by-example.html) page throws following error (Using EthereumWallet version 0.9.1 to deploy):
 
![screenshot from 2017-10-09 23-18-42](https://user-images.githubusercontent.com/1001943/31370429-5065b204-ad48-11e7-8594-b723a4f7a6e1.png)

## Error Details:

- Error Message:

```
 This declaration shadows an existing declaration.
        Voter storage delegate = voters[to];
        ^--------------------^
```

- Which version of Solidity you are using:  **0.4.11**
- What was the source code (if applicable): [Ballot Contract](https://solidity.readthedocs.io/en/latest/solidity-by-example.html)
- Which platform are you running on: Ubuntu 16.04
- How to reproduce the issue: Try to deploy Ballot contract on Rinkeyby
- What was the result of the issue: Contract didn't compile
- What the expected behaviour is: Sample code contract should compile

**Comment by axic** [comment]
Would anyone grab the relevant Mist issue (https://github.com/ethereum/mist/issues/2797) to avoid turning warnings into errors in Mist? :)

**Comment by jklepatch** [comment]
I am on it

**Comment by nisdas** [comment]
Hey I fixed the errors in the ballot contract, added public visibility identifier to the functions. The above error was due to the fact that the declaration of the struct was the same name as that of the function. So I capitalized the relevant functions. I created a pull request #3315

**Comment by kaminono** [comment]
I also encountered the same problem.

**Comment by axic** [comment]
Was fixed by #3335.
